src/chapters/more-on-types/specialtypes.rst

```diff
@@ -0,0 +1,111 @@
+Primitive Data Types
+====================
+
+.. index:: ! primitive
+
+In JavaScript, data types can fall into one of two categories: primitive or object types.
+A **primitive** data type is a basic building block.
+Using primitive data types, we can build more complex data structures or object data types.
+
+While object types such as objects and arrays are mutable, primitive data types are immutable.
+Immutable data types are data types that cannot be changed once the value has been created.
+
+Primitive data types include:
+
+1. Strings
+2. Numbers
+3. Booleans
+4. ``null``
```

Comment:
Switch the order of null and undefined, so it follows the writeup below.

Reply:
Thanks!